rama-net/src/http/request_context.rs

```diff
@@ -14,12 +14,12 @@ use rama_http_types::{Uri, Version};
 use crate::tls::SecureTransport;
 
 #[cfg(feature = "tls")]
-fn try_get_host_from_secure_transport(t: &SecureTransport) -> Option<Host> {
+fn try_get_host_from_secure_transport(t: &SecureTransport) -> Option<Domain> {
```

Comment:
can you rename these `try_get_host_*` to `try_get_sni_*` please, probably better name